libevm/VMSIMD.cpp

```diff
@@ -0,0 +1,687 @@
+	// bytes to handle "out of code" push data here.
+
+	// given the type of the vector
+	// mask and shift in the inline bytes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
+	m_SPP[0] = 0;
+	switch (t)
+	{
+	case 0:
+		for (int i = 0; i < n; ++i)
+		{
+			v8x32(m_SPP[0])[i] = m_code[++m_PC];
+		}
+		break;
+	case 1:
+		for (int i = 0; i < n; ++i)
+		{
+			uint16_t& v = v16x16(m_SPP[0])[i];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+		}
+		break;
+	case 2:
+		for (int i = 0; i < n; ++i)
+		{
+			uint32_t& v = v32x8(m_SPP[0])[i];
+			v = (v << 8) | m_code[m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+		}
+		break;
+	case 3:
+		for (int i = 0; i < n; ++i)
+		{
+			uint64_t& v = v64x4(m_SPP[0])[i];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+			v = (v << 8) | m_code[++m_PC];
+		}
+		break;
+	default: throwBadInstruction();
+	}
+}
+
+void VM::xget(uint8_t _b, uint8_t _c)
+{
+	// n type t elements in source vector, m type u in get indexes
+	const uint8_t n = nElem(_b), t = elemType(_b);
```

Comment:
Probably not.  Just a line cut and pasted into all of the other functions.  Will remove it when I split all those lines and make the const backwards.